packages/block-editor/src/layouts/flex.js

```diff
@@ -7,22 +7,44 @@ import { __ } from '@wordpress/i18n';
  * Internal dependencies
  */
 import { appendSelectors } from './utils';
+import { JustifyContentControl } from '../components/justify-content-control';
+
+const HORIZONTAL_JUSTIFY_CONTROLS = {
+	left: 'flex-start',
+	center: 'center',
+	right: 'flex-end',
+	'space-between': 'space-between',
+};
```

Comment:
This is a great addition to the flex layout, it seems we could extract it to its own smaller PR to land quickly if needed.

Reply:
You mean adding the justify control, no? I can do that but will have to use another control as the temp dropdown I put in my code now isn't the best obviously 😄

Reply:
Yes